src/main/scala/no/gdl/bookapi/model/api/Book.scala

```diff
@@ -35,7 +35,7 @@ case class Downloads(@(ApiModelProperty@field)(description = "URL to an epub-dow
 @ApiModel(description = "Information about the authors of the current book")
```

Comment:
Should probably change every "author" to "contributor" in all the descriptions of this api model (this line, and id and name below)